### PR TITLE
PHOENIX-5419: Converting anonymous to inner class in TracingQueryPlan

### PR DESCRIPTION
**Pull request by ankitjain64**

### CODE REVIEW

File: phoenix-core/src/main/java/org/apache/phoenix/compile/TraceQueryPlan.java [hunk 1]
```diff
@@ -295,4 +236,71 @@ public Long getEstimateInfoTimestamp() throws SQLException {
     public List<OrderBy> getOutputOrderBys() {
         return Collections.<OrderBy> emptyList();
     }
+
+    private class QueryResultIterator implements ResultIterator {
```
Comment (swaroopak) [inline phoenix-core/src/main/java/org/apache/phoenix/compile/TraceQueryPlan.java]: nit: could you please rename the private class to TraceQueryResultIterator?

Comment (ankitjain64) [reply phoenix-core/src/main/java/org/apache/phoenix/compile/TraceQueryPlan.java]: @swaroopak : Can you please have a look now. Thanks.

**Comment by gjacoby126** [comment]
@ankitjain64 could you please include the JIRA id PHOENIX-5419 in the commit header.

**Comment by ankitjain64** [comment]
> @ankitjain64 could you please include the JIRA id PHOENIX-5419 in the commit header.

@gjacoby126 : Done. Thanks

**Comment by gjacoby126** [comment]
@ankitjain64 - for future reference, when you squash your commits into a single one, please also edit the commit message so that it only has the one line. 

This commit message had a line for each of the former commits, and since the UI truncated it to just show the first, I didn't notice until after I pushed the merge button. I can fix it when I push to the 4.15 branches, but since this repo doesn't allow force pushes, I think it will have to remain messy in master.

**Comment by ankitjain64** [comment]
> @ankitjain64 - for future reference, when you squash your commits into a single one, please also edit the commit message so that it only has the one line.
> 
> This commit message had a line for each of the former commits, and since the UI truncated it to just show the first, I didn't notice until after I pushed the merge button. I can fix it when I push to the 4.15 branches, but since this repo doesn't allow force pushes, I think it will have to remain messy in master.

Sure, I will make sure about it from next time. Thank you.

**Pull request merged**